domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -425,32 +482,37 @@ class TopicListControllerTest {
       RATIOS_TOPIC_ID,
       RATIOS_STORY_ID_1,
       RATIOS_EXPLORATION_ID_2,
-      getCurrentTimestamp()
+      getOldTimestamp()
     )
 
-    val ongoingTopicList = retrieveOngoingStoryList()
-    assertThat(ongoingTopicList.recentStoryCount).isEqualTo(1)
-    assertThat(ongoingTopicList.olderStoryCount).isEqualTo(2)
-    verifyOngoingStoryAsFractionStory0Exploration1(ongoingTopicList.olderStoryList[0])
-    verifyOngoingStoryAsRatioStory0Exploration1(ongoingTopicList.olderStoryList[1])
-    verifyOngoingStoryAsRatioStory1Exploration3(ongoingTopicList.recentStoryList[0])
+    val promotedActivityList = retrievePromotedActivityList()
+    with(promotedActivityList.promotedStoryList) {
+      assertThat(recentlyPlayedStoryCount).isEqualTo(0)
+      assertThat(olderPlayedStoryCount).isEqualTo(3)
+
+      verifyOngoingStoryAsRatioStory0Exploration1(olderPlayedStoryList[0])
+      verifyOngoingStoryAsRatioStory1Exploration3(olderPlayedStoryList[1])
+      verifyOngoingStoryAsFractionStory0Exploration1(olderPlayedStoryList[2])
+    }
   }
 
-  private fun verifyGetOngoingStoryListSucceeded() {
+  private fun verifygetPromotedActivityListSucceeded() {
     verify(
-      mockOngoingStoryListObserver,
+      mockPromotedActivityListObserver,
       atLeastOnce()
-    ).onChanged(ongoingStoryListResultCaptor.capture())
-    assertThat(ongoingStoryListResultCaptor.value.isSuccess()).isTrue()
-  }
-
-  private fun verifyDefaultOngoingStoryListSucceeded() {
-    val ongoingTopicList = ongoingStoryListResultCaptor.value.getOrThrow()
-    assertThat(ongoingTopicList.recentStoryCount).isEqualTo(4)
-    verifyOngoingStoryAsFirstTopicStory0Exploration0(ongoingTopicList.recentStoryList[0])
-    verifyOngoingStoryAsSecondTopicStory0Exploration0(ongoingTopicList.recentStoryList[1])
-    verifyOngoingStoryAsFractionStory0Exploration0(ongoingTopicList.recentStoryList[2])
-    verifyOngoingStoryAsRatioStory0Exploration0(ongoingTopicList.recentStoryList[3])
+    ).onChanged(promotedActivityListResultCaptor.capture())
+    assertThat(promotedActivityListResultCaptor.value.isSuccess()).isTrue()
+  }
+
+  private fun verifyDefaultPromotedActivityListSucceeded() {
```

Comment:
I think this is not getting used anywhere. Remove if not needed.

Reply:
Removed.